applications/DEMApplication/python_scripts/materials_assignation_utility.py

```diff
@@ -0,0 +1,112 @@
+import KratosMultiphysics as Kratos
+from KratosMultiphysics.DEMApplication import *
```

Comment:
```suggestion
from KratosMultiphysics import DEMApplication
```

Reply:
I can't do this until I solve the 'globals()' issue. That's why I opened #8655